inc/authldap.class.php

```diff
@@ -1916,7 +1916,7 @@ static function getGroupsFromLDAP($ldap_connection, $config_ldap, $filter,
                          || in_array('ou', $groups)) {
                         $dn = $infos[$ligne][$extra_attribute];
                         $ou = array();
-                        for ($tmp=$dn ; count($tmptab=explode(',',$tmp,2))==2 ; $tmp=$tmptab[1]) {
+                        for ($tmp=$dn; count($tmptab=explode(',',$tmp,2))==2; $tmp=$tmptab[1]) {
```

Comment:
I disagree with this rules... no space between ; which ends a command.
But in for, for legibility, better to follow " ; "; typographich rule...